### PR TITLE
Set the icon for the assets.

### PR DESCRIPTION
**Pull request by annawangkkk**

### CODE REVIEW

**Comment by annawangkkk** [comment]
I set the icons for the categories. And zoom in will result in displaying more assets.

File: AssetMappr/application/showMap_cb.py [hunk 1]
```diff
@@ -46,6 +44,39 @@ def update_figure(chosen_recycling):
         # Nonlocal tells this nested function to access map_df from the outer function - otherwise throws an undefined error
         nonlocal map_df
 
+        categoryList = ["Community Centers", "Entertainment", "Financial Assistance", "Food Access",
```
Comment (mihirbhaskar) [inline AssetMappr/application/showMap_cb.py:47]: Why are we manually defining the category list here? Can't we use the 'asset_categories' list that is inputted into the showMap_cb function?

File: AssetMappr/application/showMap_cb.py [hunk 2]
```diff
@@ -46,6 +44,39 @@ def update_figure(chosen_recycling):
...
+                      'lodging',
+                      'library',
+                      'school',
+                      'school',
+                      'school',
+                      'baseball',
+                      'place-of-worship',
+                      'town-hall']
+
+        for item in zip(categoryList, colorList):
```
Comment (mihirbhaskar) [inline AssetMappr/application/showMap_cb.py]: Add a small comment here explaining what this is doing. I'm guessing this is choosing one of the categories to decide the color/symbol if there are multiple categories? Like it's choosing the first one?

File: AssetMappr/presentation/showMap.py [hunk 3]
```diff
@@ -37,7 +37,8 @@ def showMap(master_categories):
         html.Br(),
 
         # Container with the actual map, generated from a linked callback
-        dcc.Graph(id='graph', config={'displayModeBar': True, 'scrollZoom': True},
-                  style={'background': '#00FC87', 'height': '70vh', 'width': '100vh'}),
+        dcc.Graph(id='graph', config={'displayModeBar': True, 'scrollZoom': True}, style={
+                  'background': '#00FC87', 'height': '70vh', 'width': '100vh'})
+        #   ,style={'background': '#00FC87', 'height': '70vh', 'width': '100vh'}),
```
Comment (mihirbhaskar) [inline AssetMappr/presentation/showMap.py]: You can remove commented out code that we're not using - keeps the file cleaner and easier to read

**Pull request merged**